spec.html

```diff
@@ -9754,6 +9801,9 @@ <h1>SetImmutablePrototype ( _O_, _V_ )</h1>
   <emu-clause id="sec-proxy-object-internal-methods-and-internal-slots">
     <h1>Proxy Object Internal Methods and Internal Slots</h1>
     <p>A proxy object is an exotic object whose essential internal methods are partially implemented using ECMAScript code. Every proxy object has an internal slot called [[ProxyHandler]]. The value of [[ProxyHandler]] is an object, called the proxy's <em>handler object</em>, or *null*. Methods (see <emu-xref href="#table-30"></emu-xref>) of a handler object may be used to augment the implementation for one or more of the proxy object's internal methods. Every proxy object also has an internal slot called [[ProxyTarget]] whose value is either an object or the *null* value. This object is called the proxy's <em>target object</em>.</p>
+
+    <p>An object is a <dfn id="proxy-exotic-object">Proxy exotic object</dfn> if at least its essential methods (including [[Call]] and [[Construct]], if applicable) use the definitions in this section. These internal methods are installed in ProxyCreate.</p>
```

Comment:
I'm not sure what `at least` is doing here. Also, elsewhere we always say "essential _internal_ methods".

Reply:
I am also not sure what "at least" is doing there, will apply your suggestions.